pd-control.md

```diff
@@ -664,50 +684,106 @@ Usage:
->> store limit                         // Display the speed limit of adding peers in all stores
->> store limit region-add              // Display the speed limit of adding peers in all stores
->> store limit region-remove           // Display the speed limit of deleting peers in all stores
->> store limit all 5                   // Set the speed limit for all stores to add peers to 5 per minute (if no specific type is set, the default setting is the speed of adding peers)
->> store limit 1 5                     // Set the speed limit of adding peers in store 1 to 5 per minute (if no specific type is set, the default setting is the speed of adding peers)
->> store limit all 5 region-add        // Set the speed limit for all stores to add peers to 5 per minute
->> store limit 1 5 region-add          // Set the speed limit of adding peers in store 1 to 5 per minute
->> store limit 1 5 region-remove       // Set the speed limit of deleting peers in store 1 to 5 per minute
->> store limit all 5 region-remove     // Set the speed limit for all stores to delete peers to 5 per minute
->> store limit-scene                   // Display the maximum speed of adding/deleting peers in different loads (only valid when `store-limit-mode` is `auto`)
+>> store label 1 zone cn               // Set the value of the label with the "zone" key to "cn" for the store with the store id of 1
+>> store weight 1 5 10                 // Set the leader weight to 5 and region weight to 10 for the store with the store id of 1
+>> store remove-tombstone              // Remove stores that are in tombstone state
+>> store limit                         // Show limits of adding peer and removing peer operation for all stores
+>> store limit add-peer                // Show limits of adding peer operation for all stores
+>> store limit remove-peer             // Show limits of removing peer operation for all stores
+>> store limit all 5                   // Limit 5 adding peer operations and 5 remove peer operations per minute for all stores
+>> store limit 1 5                     // Limit 5 adding peer operations and 5 remove peer operations per minute for store 1
+>> store limit all 5 add-peer          // Limit 5 adding peer operations per minute for all stores
+>> store limit 1 5 add-peer            // Limit 5 adding peer operations per minute for store 1
+>> store limit 1 5 remove-peer         // Limit 5 removing peer operations per minute for store 1
+>> store limit all 5 remove-peer       // Limit 5 removing peer operations per minute for all stores
+>> store limit-scene                   // Show all limit scene (experimental)
 {
   "Idle": 100,
   "Low": 50,
   "Normal": 32,
   "High": 12
 }
->> store limit-scene idle 100          // In the scenario of setting load to idle, set the speed limit of adding/deleting peers to 100 per minute
+>> store limit-scene idle 100 // set rate to 100 in the idle scene (experimental)
```

Comment:
The original version is better?
```suggestion
>> store limit-scene idle 100 // In the scenario of setting load to idle, set the speed limit of adding/deleting peers to 100 per minute (experimental)
```

Reply:
it is a little different from the original meaning.